src/js/utils/floating-ui.js

```diff
@@ -0,0 +1,180 @@
+import merge from 'deepmerge';
+import { shouldCenterStep } from './general';
+import {
+  computePosition,
+  autoUpdate,
+  shift,
+  arrow,
+  limitShift
+} from '@floating-ui/dom';
+
+/**
+ * Floating UI Options
+ *
+ * @typedef {object} FloatingUIOptions
+ */
+
+/**
+ * Determines options for the tooltip and initializes
+ * `step.tooltip` as a Popper instance.
+ * @param {Step} step The step instance
+ */
+export function setupTooltip(step) {
+  if (step.cleanup) {
+    step.cleanup();
+  }
+
+  const attachToOptions = step._getResolvedAttachToOptions();
+
+  let target = attachToOptions.element;
+  const floatingUIOptions = getFloatingUIOptions(attachToOptions, step);
+
+  if (shouldCenterStep(attachToOptions)) {
```

Comment:
default behavior of floatingui is to center the tooltip, no need to do anything specific here.